### PR TITLE
fix no machine version

### PR DESCRIPTION
**Pull request by ipa-fmw**

### CODE REVIEW

File: images_config/ks-robot-master.cfg [hunk 1]
```diff
@@ -271,11 +271,15 @@ chmod +x /usr/local/bin/socket_buffer.py
 #initctl start candump_any
 %end
 
-################ INSTALL NOMACHINE ################ 
+################ INSTALL PULSEAUDIO ################ 
 %post --erroronfail --interpreter /bin/bash
 apt-get install pulseaudio -y
```
Comment (ipa-fxm) [inline images_config/ks-robot-master.cfg]: should packages to be installed not be listed in [packages section](https://github.com/ipa-fmw/setup_cob4/blob/4e2bc5487e07f189d567cbbeb70e1958d9b126a7/images_config/ks-robot-master.cfg#L45) further up?

File: images_config/ks-robot-master.cfg [hunk 2]
```diff
@@ -271,11 +271,15 @@ chmod +x /usr/local/bin/socket_buffer.py
...
-NOMACHINE_VERSION=5.3.12_10
-wget -O /root/nomachine_${NOMACHINE_VERSION}_amd64.deb http://download.nomachine.com/download/5.3/Linux/nomachine_${NOMACHINE_VERSION}_amd64.deb
+%end
+
+################ INSTALL NOMACHINE ################ 
+%post --erroronfail --interpreter /bin/bash
+NOMACHINE_VERSION=6.0.62_6
+wget -O /root/nomachine_${NOMACHINE_VERSION}_amd64.deb http://download.nomachine.com/download/6.0/Linux/nomachine_${NOMACHINE_VERSION}_amd64.deb
 dpkg -i /root/nomachine_${NOMACHINE_VERSION}_amd64.deb
```
Comment (ipa-fxm) [inline images_config/ks-robot-master.cfg]: maybe the same can be done for `nomachine`, too
like so `nomachine=NOMACHINE_VERSION` (see https://askubuntu.com/a/428778)?

Comment (ipa-fmw) [reply images_config/ks-robot-master.cfg]: the problem is that nomachine is not available through the ubuntu package repository. so we need to manually download the deb file and install it.
Obviously nomachine does not provide older version on their servers any more, so we have to update to a more recent version manually

File: images_config/ks-robot-master.cfg (hunk 1 above)
Comment (ipa-fmw) [reply images_config/ks-robot-master.cfg]: done in https://github.com/ipa320/setup_cob4/pull/192/commits/34ad7962cec0444bb1500e24a25f335f95e3d251

**Comment by ipa-fmw** [comment]
all comments adressed

**Pull request merged**